learn/cookbooks/multitenancy_nodejs.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: How to implement multitenancy with Node.js — Meilisearch documentation
+description: Learn how to build secure, multi-tenant search in your Node.js application.
+sidebarDepth: 3
+---
+
+# How to implement multitenancy with Node.js
+
+## What is multitenancy?
+
+In software development, multitenancy means that multiple users — also called tenants — share the same computing resources but have different access to system-wide data.
```

Comment:
Not sure if this is really a problem, but it's the exact same sentence we use in the multitenancy and tenant tokens section of the documentation